front/src/modules/trainschedule/components/Timetable/types.ts

```diff
@@ -44,3 +44,11 @@ export type InvalidReason =
   | Extract<SimulationSummaryResult['status'], 'pathfinding_failure' | 'simulation_failed'>
   | PathfindingNotFound['error_type']
   | PathfindingInputError['error_type'];
+
+export type PacedTrainResultWithTrainId = Omit<TrainScheduleWithDetails, 'id'> & {
```

Comment:
To harmonize with TrainScheduleWithDetails
```suggestion
export type PacedTrainWithResult = Omit<TrainScheduleWithDetails, 'id'> & {
```

Reply:
done